Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/RemoteXDAInstance.tsx

```diff
@@ -65,21 +67,12 @@ function RemoteXDAInstance(props: IProps) {
             return 'systemSettings';
     }
 
-    function returnMain() {
-        history.push({ pathname: homePath + 'index.cshtml', search: '?name=RemoteXDAInstanceMain', state: {} })
-    }
-
     function deleteConnection(): void {
         setLoading(true);
-        if (props.Roles.includes('Administrator')) dispatch(RemoteXDAInstanceSlice.DBAction({ verb: 'DELETE', record: connection }));
-        setLoading(false);
-        returnMain();
-    };
-
-    function patchConnection(connection: OpenXDA.Types.RemoteXDAInstance): void {
-        setLoading(true);
-        dispatch(RemoteXDAInstanceSlice.DBAction({ verb: 'PATCH', record: connection }));
+        if (props.Roles.includes('Administrator'))
+            dispatch(RemoteXDAInstanceSlice.DBAction({ verb: 'DELETE', record: connection }));
         setLoading(false);
+        window.location.href = homePath + 'index.cshtml?name=RemoteXDAInstanceMain';
```

Comment:
As per other conversation, does this resolve the issue?

Reply:
Yes I used the Customer page for reference and this was the only difference I could see. With this change it seems to remove the problem